### PR TITLE
Create Headings README.md

### LINKED ISSUE #915: Create Headings component Readme.md
**Is your feature request related to a problem? Please describe.**
We currently don't have explanations of what each component does, how it should be used, how it shouldn't be used.

We should add these explanations for the Headings component. 

This is a prerequisite to moving components to an external component directory. 

**Describe the solution you'd like**
We should have a `README.md` file in the `src/app/components/Headings/` directory. 
It should have these sections.

```
- ## Description (high-level summary that will be accessible to product/UX/Test, plus link to Storybook, explain the different cases)
- ## When to use this component (including where it is currently used)
- ## When to not use this component (if this is not known, can leave the section empty)
- ## Accessibility notes (using accessiblity notes on Zeplin https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/5babb3dea83a3937890e374c, ensuring they make sense to non-technical audiences)
- ## Roadmap (what we have planned for this component)
- ## Additional notes (link to any relevant ADRs)
```

**Describe alternatives you've considered**
We have considered other sections in the readme, but these seem like a good starting point. We can iterate/add/change sections in future, if needed. 

**Testing notes**
The `README.md` file should be clear, understandable, to people who are not developers and give a good explanation about the scope of a component. 

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #915

Adds a readme to the headings component

- ~[ ] Tests added for new features~
- ~[ ] Test engineer approval~

### CODE REVIEW

File: src/app/components/Headings/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Headings [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=Headline)
+
+## Description
+
+The Headings are a set of two components, `Headline` and `Subheading`. They use a `h1` and `h1` HTML element respectively, with the `Headline` being designed for a single use on the page, with `Subheading` being aimed at repeate use.
```
Comment (bcmn) [inline src/app/components/Headings/README.md]: > They use a `h1` and `h1` HTML element respectively

I assume the second of these is incorrect? 😛

> being aimed at repeate use.

Similarly, _repeated_.

Comment (bcmn) [inline src/app/components/Headings/README.md]:
```suggestion
The Headings are a set of two components, `Headline` and `Subheading`. They use a `h1` and `h2` HTML element respectively, with the `Headline` being designed for a single use on the page, with `Subheading` being aimed at repeated use.
```

File: src/app/components/Headings/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## When to use this component
+
+These components can be used at any point on the page, however the `Headline` is designed to be used once at the top of the page. The `Subheadline` adds an `id` value to the `h2` which can be used as an ancor when referencing content.
```
Comment (bcmn) [inline src/app/components/Headings/README.md]:
```suggestion
These components can be used at any point on the page, however the `Headline` is designed to be used once at the top of the page. The `Subheadline` adds an `id` value to the `h2` which can be used as an anchor when referencing content.
```

**Review by bcmn — CHANGES_REQUESTED** [review]

**Review by bcmn — APPROVED** [review]

**Review by jtart — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
👍

**Pull request merged**